.github/workflows/proto_filepull.yml

```diff
@@ -0,0 +1,50 @@
+---
+name: "Flat"
+
+on: # yamllint disable-line rule:truthy
+  schedule:
+    - cron: "0 8 * * 4"
+
+jobs:
+  data_gathering:
+    runs-on: "ubuntu-latest"
+    env:
+      BRANCH_NAME: "PROTO_Updates"
+    steps:
+      # Checkout repo
```

Comment:
I'd check on https://github.com/networktocode/netutils/pull/197 to make sure we follow suit here.